Makefile

```diff
@@ -413,6 +452,6 @@ set -e; \
 package=$(2)@$(3) ;\
 echo "Downloading $${package}" ;\
 GOBIN=$(LOCALBIN) go install $${package} ;\
-mv "$$(echo "$(1)" | sed "s/-$(3)$$//")" $(1) ;\
+if [ ! -f $(1) ]; then mv -f "$$(echo "$(1)" | sed "s/-$(3)$$//")" $(1); fi ;\
```

Comment:
GitHub actions, there's a commit back in the history that describes why:

```
When GitHub Actions runs the 'make cli-install' target it fails
because the GOBIN is already set to ./bin.  GitHub appears to
use a 'mv' that by default does not wish to clobber the file
that now exists (even with -f specified), so place a check in
front of the 'mv' to prevent it from ever running if the file
exists there.  This does not change the existing behavior of
the make target as it does not run if a file exists anyways.
```

Reply:
Ok got it Actions quirks